### ISSUE TITLE
SA1004 false-positive when first token on XML doc line is out keyword from wrapped params in <see cref="...."/>

### ISSUE THREAD

**Issue by mikernet** [DotNetAnalyzers/StyleCopAnalyzers#3817]
Example:
```cs
/// <summary>
/// This is a summary.
/// </summary>
/// <remarks>
/// <para>
/// Description of some remarks that refer to a method: <see cref="SomeMethod(int, int,
/// out string)"/>.</para>
/// </remarks>
public class SomeClass
{
    public void SomeMethod(int x, int y, out string z) => z = string.Empty;
}
```
![image](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/assets/1700111/e6594d76-8688-4479-931a-01a10b7a358e)
